assignment2/hex_calculator/Calculation.java

```diff
@@ -0,0 +1,133 @@
+import java.util.Scanner;
+public class Calculation 
+{
+	private String hexNumber1, hexNumber2,outputHexNumber;
+	private int intNumber1,intNumber2,outputIntNumber;
+	
+	void welcomeScreen()
```

Comment:
Should add access modifiers to functions. In real projects, there has many packages where access modifiers will prove important.